### DISCUSSION TITLE
adding a new connector (MindsDB)

### DISCUSSION THREAD

**Discussion by torrmal** [dbeaver/dbeaver#16108] [DBeaver Development] [answered]
Hi we are looking to add a new connector to MindsDB, we are happy to pay for this work, or make a donation, do you know with whom can we speak, it should be about cloning the MariaDB connector, as we speak the MySQL wire protocol, but there are parts that we would like to customize.

**Comment by tati-kru** [comment, marked as answer]
Please, contact DBeaver Enterprise support (support@dbeaver.com) for these questions.
